docs/src/main/resources/Extending.adoc

```diff
@@ -1,5 +1,5 @@
 = Extending {branding} Platform Application
-include::config.adoc[]
+include::../includes/config.adoc[]
```

Comment:
Talked to @ricklarsen but these includes have relative paths that shouldn't be there